repository_service_tuf_worker/interfaces.py

```diff
@@ -48,15 +49,8 @@ def settings(cls) -> List[ServiceSettings]:
         pass  # pragma: no cover
 
     @abstractmethod
-    def get(self, rolename: List[str]) -> Dict[str, Any]:
-        """Return a key from specific rolename."""
-        pass  # pragma: no cover
-
-    @abstractmethod
-    def put(self, file_object: str, filename: str) -> None:
-        """
-        Stores file object with a specific filename.
-        """
+    def get_signer(self) -> Signer:
```

Comment:
```suggestion
    def get(self) -> Signer:
```

We don't need to rename the existing method.
If we define that returns, a Singer makes ok. 
IMHO, keep the name `get` aligned with the other Interface (`IStorage`) with the method `get`.

As it is an API (GET, PUT, DELETE, etc.) will look better.

Reply:
Okay, I don't have a strong opinion on this one, so I will return it to `get()`.